charts/rancher-vsphere-csi/templates/node/role-binding.yaml

```diff
@@ -12,3 +12,18 @@ roleRef:
   kind: Role
   name: vsphere-csi-node-role
   apiGroup: rbac.authorization.k8s.io
+
+---
+
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: vsphere-csi-node-cluster-role-binding
+subjects:
+  - kind: ServiceAccount
+    name: vsphere-csi-node
+    namespace: vmware-system-csi
```

Comment:
should be `namespace: {{ .Release.Namespace }}`

Reply:
Made the required changes.